src/main.cc

```diff
@@ -192,41 +191,41 @@ static int daemon_main(LiteClient& client, const bpo::variables_map& variables_m
     client.reportHwInfo();
 
     try {
-      auto target = find_target(client, hwid, client.tags, "latest"); // target cannot be nullptr, an exception will be yielded if no target
-
-
-    // This is a workaround for finding and avoiding bad updates after a rollback.
-    // Rollback sets the installed version state to none instead of broken, so there is no
-    // easy way to find just the bad versions without api/storage changes. As a workaround we
-    // just check if the version is known (old hash) and not current/pending and abort if so
-    bool known_target_sha = known_local_target(client, *target, installed_versions);
-    if (!known_target_sha && ! client.isTargetCurrent(*target)) {
-      LOG_INFO << "Got a New Target !";
-
-      data::ResultCode::Numeric rc = do_update(client, *target);
-      if (rc == data::ResultCode::Numeric::kOk) {
-        current = *target;
-        client.http_client->updateHeader("x-ats-target", current.filename());
-        // Start the loop over to call updateImagesMeta which will update this
-        // device's target name on the server.
-        continue;
-      } else if (rc == data::ResultCode::Numeric::kNeedCompletion) {
-        // no point to continue running TUF cycle (check for update, download, install)
-        // since reboot is required to apply/finalize the currently installed update (aka pending update)
+      auto target = find_target(client, hwid, client.tags,
+                                "latest");  // target cannot be nullptr, an exception will be yielded if no target
```

Comment:
That's really weird formatting TBH, is it what clang-format wants? Maybe put the comment above the `find_traget` statement (just minor, not important)

Reply:
good idea.